guidelines/README.md

```diff
@@ -1,12 +1,26 @@
 # Guidelines
 
-This part describes guidelines for general usage of the Connector API and serves as a quick-start guide to kickstart your integration development.
+This section describes guidelines for general usage of the __Mews Connector API__ and serves as a quick-start guide to kickstart your integration development.
```

Comment:
By specifically using the word 'Mews' and putting it in bold, I am branding the Connector API as part of the Mews Open API, this is a convention I have been adopting in all documentation and I've even put it in the Style Guide. IronicallyI do agree that there is confusion in having two similar products with almost the same name, and we really ought to do something about that, but that's a separate problem that already existed. That can only be fixed by changing the name of at least one of the two products - I shall raise the issue with Jord and see what we can do.

Reply:
By specifically using the word 'Mews' and putting it in bold, I am branding the Connector API as part of the Mews Open API, this is a convention I have been adopting in all documentation and I've even put it in the Style Guide. Ironically, part of the reason for doing this is to give it its own identity and avoid any confusion! I do agree that there is confusion in having two similar products with almost the same name, and we really ought to do something about that, but that's a separate problem that already existed. That can only be fixed by changing the name of at least one of the two products - let me raise the issue and see what we can do, because that's the real solution.